config/runtime.exs

```diff
@@ -315,3 +315,7 @@ config :bot_manager, :end_point_configuration,
   scheme: :http,
   plug: BotManager.Endpoint,
   options: [port: bot_manager_port]
+
+###################################
+
+config :joken, issuer: "https://accounts.google.com", audience: System.get_env("GOOGLE_CLIENT_ID")
```

Comment:
Testing this is local the issuer was wrong, for some reason it expects "accounts.google.com" without the https, i replaced it in the code and could create a google user 🥳

Reply:
This was an issue related to the way i generated the token